Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM inclusivedesign/nodejs:0.10.33
+ 
+RUN yum -y install git && \
+yum clean all
+ 
+RUN mkdir -p /opt/universal
```

Comment:
If we put our code under "opt" we won't be able to run the tests inside the container because node won't find the module 'universal'. To fix that, I'd say that we should put 'universal' as '/opt/gpii/node_modules/universal'.